.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,24 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'status: needs triage, type: bug'
+assignees: ''
+
+---
+
 <!--
 If you encounter a bug or you have a support question, please try to fill out some of the information below.
-Generally speaking, the information below is meant to help debugging issues but is no prerequisite for opening an issue.
+Please take a look at the [troubleshooting guide](https://haskell-language-server.readthedocs.io/en/latest/troubleshooting.html) before filing a new issue.
+The information below is meant to help debugging issues but is no prerequisite for opening an issue.
 -->
 
 ### Your environment
-Output of `haskell-language-server --probe-tools` or `haskell-language-server-wrapper --probe-tools`:
```

Comment:
This is still useful, no?

Reply:
`haskell-language-server --debug .` also prints the tools, so I think this is indeed superfluous.